src/AutoRest.CSharp/Mgmt/Output/Resource.cs

```diff
@@ -484,7 +484,7 @@ protected virtual Method BuildCreateResourceIdentifierMethod()
             var signature = new MethodSignature(
                 Name: "CreateResourceIdentifier",
                 null,
-                Description: $"Generate the resource identifier of a <see cref=\"{Type.ToStringForDocs()}\"/> instance.",
```

Comment:
I assume this method `CSharpType.ToStringForDocs` is no longer needed after this change? Could we remove it?

Reply:
Done